### ISSUE TITLE
QuestieQuest error

### ISSUE THREAD

**Issue by silverhawke249** [AeroScripts/QuestieDev#1457]
## Bug description
Error line appears after taking the quest The Love Potion from Nagmara in Blackrock Depths.
```
Questie: [ERROR] [QuestieQuest]: There was an error populating objectives for The Love Potion 4201 1
Interface\AddOns\Questie\Modules\QuestieQuest.lua:931: attempt to index field '?' (a nil value)
```

## Screenshots
![image](https://user-images.githubusercontent.com/50395934/67657224-20f4d080-f991-11e9-9aae-98886f1edf06.png)

## Questie version
Version: 5.0.0 7 fb7c60e

**Comment by laurencei** [comment]
Can confirm - have the same bug for that quest on my main.

Also got the exact same error for "Ak'Zeloth 809 1"it on an alt.

**Comment by drejjmit** [comment]
Fixed in 5.0.1. Please update.

**Comment by silverhawke249** [comment]
Can confirm that it's fixed. Thank you!
